packages/auth/src/features/system/handler.test.ts

```diff
@@ -26,11 +26,7 @@ describe('authenticate handler receives a request', () => {
       fetch.mockReject(new Error())
       const res = await server.server.inject({
         method: 'POST',
-        url: '/authenticateSystemClient',
-        payload: {
-          client_id: '123',
-          client_secret: '456'
-        }
+        url: '/token?client_id=123&client_secret=456&grant_type=client_credentials'
```

Comment:
Is it required for the client_id & client_secret to be passed as search params? It's not even a GET request

Reply:
hm I guess that's the standard

Reply:
Yeah, assuming that's the standard as it's exampled [here](https://www.oauth.com/oauth2-servers/access-tokens/client-credentials/)